packages/manager/apps/pci-workflow/src/pages/new/utils/is-snapshot-consumption.ts

```diff
@@ -0,0 +1,4 @@
+export const isSnapshotConsumption = (
+  code: string | null | undefined,
+): boolean =>
+  typeof code === 'string' && code.startsWith('snapshot.consumption');
```

Comment:
Could be simplified: 
```suggestion
export const isSnapshotConsumption = (code?: string | null): boolean =>
  code?.startsWith('snapshot.consumption');
```